src/main/java/coresearch/cvurl/io/model/CVurlConfig.java

```diff
@@ -168,6 +171,16 @@ public void setIsLogEnable(boolean enabled) {
         this.getGlobalRequestConfiguration().setLogEnabled(enabled);
     }
 
+    /**
+     * Creates instance of {@link HttpClient} with default settings except ProxySelector
+     * @return HttpClient with CVurlProxySelector
+     */
+    static HttpClient makeDefaultHttpClientWithCVurlProxySelector() {
```

Comment:
Why it isn't private?

Reply:
It was used in a test. Made it private and updated test.